beamer/cli.py

```diff
@@ -103,6 +104,9 @@ def main(
 ) -> None:
     beamer.util.setup_logging(log_level=log_level.upper(), log_json=False)
 
+    version = pkg_resources.get_distribution("beamer-bridge").version
```

Comment:
Nitpick: since we rely on a recent-enough version of Python, I think we should use `importlib.metadata.version` here, that stuff is intended to replace functionality offered by the old `pkg_resources` package (see https://docs.python.org/3/library/importlib.metadata.html).

Reply:
How do you remember such things 🤯